### PR TITLE
added file breakdown table

### PR DESCRIPTION
**Pull request by nyanofthemoon**

### CODE REVIEW

File: src/components/CohortBuilder/Summary/index.js [hunk 1]
```diff
@@ -44,11 +50,22 @@ const sortDescParticipant = (a, b) => {
...
     overflow: visible;
   }
 `;
 
+const CardSlotOverflowScroll = styled(Card)`
+  height: 305px;
+  & div {
+    overflow: scroll;
+  }
+`;
```
Comment (hlminh2000) [inline src/components/CohortBuilder/Summary/index.js]: The card should have some build-in mechanism for scrolling, which comes with a little fade effect that goes away when at bottom. You can refer to the Saved Queries card in the dashboard page :) 
screen shot 2019-02-08 at 5 01 33 pm

Comment (nyanofthemoon) [reply src/components/CohortBuilder/Summary/index.js]: Ok, fixed!

**Comment by nyanofthemoon** [comment]
@rosibaj @hlminh2000 Satisfactory?

File: src/components/CohortBuilder/Summary/FileBreakdown.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  return `${Number(quantity).toLocaleString()} file${quantity > 1 ? 's' : ''}`;
+};
+
+const generateFileColumnContents = dataset => {
+  return dataset.map(datum => {
+    datum.fileLink = (
+      <a href={generateFileRepositoryUrl(datum.dataType, datum.experimentalStrategy)}>
+        {localizeFileQuantity(datum.files)}
+      </a>
+    );
```
Comment (hlminh2000) [inline src/components/CohortBuilder/Summary/FileBreakdown.js]: let's not mutate data in map. The following is preferred:
```
return {
    ... datum,
    fileLink: (
        <a href={generateFileRepositoryUrl(datum.dataType, datum.experimentalStrategy)}>
            {localizeFileQuantity(datum.files)}
        </a>
    )
}
```

Comment (nyanofthemoon) [reply src/components/CohortBuilder/Summary/FileBreakdown.js]: Done'd. @hlminh2000 Please approve.

File: src/components/CohortBuilder/Summary/FileBreakdown.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        columns={[
+          { Header: 'Data Type', accessor: 'dataType' },
+          { Header: 'Experimental Strategy', accessor: 'experimentalStrategy' },
+          { Header: 'Files', accessor: 'fileLink' },
+        ]}
+        data={finalData}
+        transforms={{
+          dataType: dataType => <Column>{dataType}</Column>,
+          experimentalStrategy: experimentalStrategy => <Column>{experimentalStrategy}</Column>,
+          fileLink: fileLink => <FilesColumn>{fileLink}</FilesColumn>,
```
[the remark this answers is not in the document]
Comment (nyanofthemoon) [reply src/components/CohortBuilder/Summary/FileBreakdown.js]: ...yes.

**Pull request merged**